App/Controllers/IndexController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Controllers;
+
+use Core\View;
+use App\Models\User;
+use App\Models\Task;
+
+class IndexController {
+
+    public function index()
+    {
+        if(session_status() !== PHP_SESSION_ACTIVE)
+            session_start();
+
+        $tasks=[];
+        $teamMembers=[];
+        if (isset($_SESSION["id"]))
+        {
+            $sqlParams['user_id']=$_SESSION["id"];
+            $tasks = Task::getDB()->read(Task::getTableName(),$sqlParams,Task::class);
+
+            $user = User::getDB()->readById(User::getTableName(),$_SESSION["id"],User::class);
```

Comment:
As a standarization, we tend to use a variable to set the value of $_SESSION['id'] to avoid any mispells.